### PR TITLE
Adding positive_min and strict_positive_min to stats endpoint

### PR DESCRIPTION
**Pull request by loichuder**

### CODE REVIEW

File: test/base_test.py [hunk 1]
```diff
@@ -129,6 +129,7 @@ def test_stats_on_array(self, server):
         image[0, 1] = np.inf
         finite_h5data = image[np.isfinite(image)]
         expected_stats = {
+            "positive_min": np.min(finite_h5data[finite_h5data > 0]),
             "min": np.min(finite_h5data),
             "max": np.max(finite_h5data),
             "mean": np.mean(finite_h5data),
```
Comment (axelboc) [inline test/base_test.py:135]: This test reimplements the code it's testing 😁 - might be better to create an array with known values and hard-code the positive min, min, max and mean. 🤷

**Review by axelboc — APPROVED** [review]

File: h5grove/content.py [hunk 2]
```diff
@@ -114,6 +115,7 @@ def data_stats(
 
         cast = float if np.issubdtype(data.dtype, np.floating) else int
         return {
+            "positive_min": cast(np.min(data[data > 0])),
```
Comment (t20100) [inline h5grove/content.py:118]: What about using `strictly_positive_min` (or `min>0`) naming to leave room for `min>=0`?

**Review by t20100 — APPROVED** [review]

**Comment by loichuder** [comment]
Rerequesting rereviews as I added `strict_positive_min` and strengthened the tests

**Review by axelboc — APPROVED** [review]

**Review by t20100 — APPROVED** [review]
LGTM

**Pull request merged**